src/state/store.ts

```diff
@@ -119,13 +123,20 @@ export const useRecipeStore = create<RecipeStore>()(
 
             set({
                 selectedRecipeId: recipeId,
+                resultUrl: SIMULARIUM_EMBED_URL + (sel.result_path ?? ""),
```

Comment:
you're storing redundant information in the store. if we already know the result path, we can construct it without having to store it separately here